db/migrations/20170915150351_drop_title_from_movies.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+exports.up = function (Knex, Promise) {
+  return Knex.schema.table('movies', (table) => {
+    table.dropColumn('title');
+  })
+  .then(() => {
+    return Knex.raw('ALTER TABLE movies ADD CONSTRAINT movies_name_not_null CHECK (name IS NOT NULL) NOT VALID');
```

Comment:
You've got the right idea here to first add the constraint without validating it. However, the next step is another line to go through and validate it. So something like:
`.then(() => Knex.raw('ALTER TABLE movies VALIDATE CONSTRAINT movies_name_not_null'))`

Reply:
Thanks. I've added the validation now.